spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -1273,55 +1288,15 @@ dataflow:>task destroy --name task1
 
 === Application and Server Properties
 
-This section covers how you can customize the deployment of your applications. You can use a number of properties to influence settings for the applications that are deployed. Properties can be applied on a per-application basis or in the server configuration for all deployed applications.
+This section covers how you can customize the deployment of your applications. You can use a number of properties to influence settings for the applications that are deployed. Properties can be applied on a per-application basis or in the appropriate server configuration for all deployed applications.
 
 NOTE: Properties set on a per-application basis always take precedence over properties set as the server configuration. This arrangement allows for the ability to override global server level properties on a per-application basis.
 
-See https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/blob/master/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java[`KubernetesDeployerProperties`] for more on the supported options.
-
-==== Using Deployments
-
-The deployer uses `ReplicationController` by default. To use deployments instead, you can set the following option as part of the container `env` section in a deployment YAML file:
-
-====
-[source,yaml]
-----
-env:
-- name: SPRING_CLOUD_DATAFLOW_TASK_PLATFORM_KUBERNETES_ACCOUNTS[default]_CREATE_DEPLOYMENT
-  value: 'true'
-----
-====
-
-This is now the preferred setting and will be the default in future releases of the deployer.
+Properties to be applied for all deployed Tasks are defined in the `src/kubernetes/server/server-config-(binder).yaml` file and for Streams in `src/kubernetes/skipper/skipper-config-(binder).yaml`. Replace `(binder)` with the messaging middleware you are using, for example `rabbit` or `kafka`. By default properties are defined in the `application.yaml` configuration block under the namespace `spring.cloud.dataflow.task.platform.kubernetes.accounts.default` for Tasks and `spring.cloud.skipper.server.platform.kubernetes.accounts.default` for Streams.
```

Comment:
There are 2 levels of properties we describing here, but the latter part (i.e: "By default properties are defined...") can be clarified as to what properties are specifically listed in application.yaml.